api/src/stampy_chat/get_blocks.py

```diff
@@ -132,7 +132,7 @@ def get_top_k_blocks(index, user_query: str, k: int) -> List[Block]:
         group = list(group)
         if len(group) == 0: continue
 
-        group = group[:3] # limit to a max of 3 blocks from any one source
+        # group = group[:3] # limit to a max of 3 blocks from any one source
```

Comment:
@FraserLee do you remember why you set this limit at the time? I also agree to not having the limit